CountryUtils.ts

```diff
@@ -21,7 +21,7 @@ export class CountryUtils {
                 (item: CountryCode) : Country => CountryUtils.createCountryByCode(item)
             );
         }
-        return this._countryList;
+        return this._countryList!;
```

Comment:
It's much better to throw an exception here since `this._countryList` may really be `undefined`.

E.g.  add a line like this before the return:

```typescript
if (!this._countryList) throw TypeError(`CountryUtils: Country list not initialized`);
```

Same problem is with .getCountryCodeList(), so same there, too.